apps/omg_status/lib/omg_status/application.ex

```diff
@@ -39,22 +40,33 @@ defmodule OMG.Status.Application do
       else
         [
           {OMG.Status.Monitor.StatsdMonitor, [alarm_module: Alarm, child_module: Datadog]},
-          {OMG.Status.Monitor.MemoryMonitor,
-           [
-             alarm_module: Alarm,
-             memsup_module: :memsup,
-             threshold: system_memory_high_threshold,
-             interval_ms: system_memory_check_interval_ms
-           ]},
+          {
+            OMG.Status.Monitor.MemoryMonitor,
+            [
+              alarm_module: Alarm,
+              memsup_module: :memsup,
+              threshold: system_memory_high_threshold,
+              interval_ms: system_memory_check_interval_ms
+            ]
+          },
+          {
+            Telemetry,
+            [
+              release: Application.get_env(:omg_status, :release),
+              current_version: Application.get_env(:omg_status, :current_version)
```

Comment:
you can add these two into `OMG.Status.Configuration` and access them via this module

Reply:
✅